rocket.py

```diff
@@ -0,0 +1,33 @@
+import pygame
+import os 
+from menu import *
+#First let's make a moving rectangle here and then modify to make a moving rocket
+class Rocket():
+    def __init__(self, game):
+        self.game = game #Use a game object
+        self.width, self.height = self.game.DISPLAY_W / 20, self.game.DISPLAY_H / 20#Size of the rectangle
```

Comment:
nit: try to keep the aspect ratio

```suggestion
        self.width, self.height = self.game.DISPLAY_W / 20, self.game.DISPLAY_H / 20# Size of the rectangle
```

Reply:
yes, I fixed it.